indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -49,21 +60,46 @@ export default function EditableList() {
       codePresent={codePresent}
       editableType={type}
       eventId={eventId}
+      editors={editors}
     />
   );
 }
 
-function EditableListDisplay({editableList, codePresent, editableType, eventId}) {
+function EditableListDisplay({
+  editableList: origEditableList,
+  codePresent,
+  editableType,
+  eventId,
+  editors,
+}) {
   const [sortBy, setSortBy] = useState('friendly_id');
   const [sortDirection, setSortDirection] = useState('ASC');
+  const [editableList, setEditableList] = useState(origEditableList);
   const [sortedList, setSortedList] = useState(editableList);
```

Comment:
Why do we need both `editableList` and `sortedList`? Wouldn't just using the sorted for the same work?

Reply:
Yes, most likely that would be possible by using `useMemo` for the sorted list instead of putting it in state. But I did not want to refactor this component in this PR. Also because the tricky part with useMemo is invalidating it after patching the data with updated entries.